composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/event-dispatcher": "^5.4 || ^6.0",
         "symfony/form": "^5.4 || ^6.0",
         "symfony/http-foundation": "^4.4 || ^5.0.7",
-        "symfony/http-kernel": "^4.4 || ^5.1.5",
+        "symfony/http-kernel": "^4.4 || ^5.1.5 || ^6.0",
```

Comment:
Could you change both of these to `^5.4 || ^6.0`? Thanks

I must have forgotten them on the recent update

Reply:
Done!